begin
  a
rescue Exception => ex
  b
ensure
  b
end
